net/mwan3/files/lib/mwan3/mwan3.sh

```diff
@@ -969,17 +973,17 @@ mwan3_set_user_iptables_rule()
 
 			fi
 		fi
-
-		if [ "$family" == "any" ]; then
-
-			for IPT in "$IPT4" "$IPT6"; do
-				case $proto in
-					tcp|udp)
+		for IPT in "$IPT4" "$IPT6"; do
+			[ "$family" == "ipv4" ] && [ "$IPT" == "$IPT6" ] && continue
+			[ "$family" == "ipv6" ] && [ "$IPT" == "$IPT4" ] && continue
+			case $proto in
+				tcp|udp)
 					[ "$global_logging" = "1" ] && [ "$rule_logging" = "1" ] && {
 						$IPT -A mwan3_rules \
 							-p $proto \
-							-s $src_ip \
-							-d $dest_ip $ipset \
+							${src_ip+-s} $src_ip \
```

Comment:
I think there's a colon missing?

Reply:
good catch - it actually works without one, which is why I missed it, but it should be there.